docs/fundamentals/diagnostics/runtime-tiered-compilation-events.md

```diff
@@ -0,0 +1,117 @@
+
+## TieredCompilationPause event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationPause`|281|Tiered compilation was paused. Tiered compilation may be paused due to startup-like activities such as a new method being called for the first time, to reduce overhead during application startup, such as call-counting and background JIT-compilation. It would resume after startup-like activities cease for a short duration.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+
+## TieredCompilationResume event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationResume`|282|Tiered compilation was resumed.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+|`NewMethodCount`|`win:UInt32`|The number of new methods called for the first time while tiered compilation was paused.|
+
+## TieredCompilationBackgroundJitStart event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationBackgroundJitStart`|283|Background JIT-compilation has started.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+|`PendingMethodCount`|`win:UInt32`|The number of methods pending background JIT-compilation.|
+
+## TieredCompilationBackgroundJitStop event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationBackgroundJitStop`|284|Background JIT-compilation has stopped.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+|`PendingMethodCount`|`win:UInt32`|The number of methods still pending background JIT-compilation.|
+|`JittedMethodCount`|`win:UInt32`|The number of methods that were JIT-compiled in the background since background JIT-compilation had last started.|
```

Comment:
```suggestion
|`JittedMethodCount`|`win:UInt32`|The number of methods that were JIT-compiled in the background since background JIT-compilation last started.|
```